docs/Developers/Development_Setup.rst

```diff
@@ -43,14 +43,25 @@ Using virtualenv
 - Install virtualenv using pip :
     ::
 
-        $ pip install virtualenv
+        $ pip3 install virtualenv
 
 - Create the virtualenv :
     ::
 
         $ cd working_dir # move into the dir where you want to create coala-venv
         $ virtualenv coala-venv
 
+-NOTE:
+ If you have both Python 3 and Python 2 installed check this out first.
```

Comment:
"check this out" -> "try this command"
"check this out" seems to be asking the user to check the output which is not the case here.
Also checkout has been overloaded in open source to also mean switch branch so it would also be confusing.